src/applications/mhv/medical-records/tests/e2e/medical-records-download-txt-functionality-all.cypress.spec.js

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment-timezone';
+import MedicalRecordsSite from './mr_site/MedicalRecordsSite';
+
+describe('Medical Records Download All TXT Functionality', () => {
+  it('Medical Records Download All TXTX Functionality', () => {
+    const site = new MedicalRecordsSite();
+    site.login();
+    // Given Navigate to Radiology Page
+    cy.visit('my-health/medical-records/download-all');
+
+    cy.get('[data-testid="download-blue-button-txt"]').click();
+    // cy.readFile(`${Cypress.config('downloadsFolder')}/radiology_report.pdf`);
+    site.verifyDownloadedPdfFile(
```

Comment:
Did you intended for this line to call the "verifyDownloadedTxtFile" function instead of the "verifyDownloadedPdfFile" function?

Reply:
Fixed